.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
 public/
-
-test/test.js
+dist/
```

Comment:
I'm not sure if `dist` should be ignored. If someone pulls this package in via different means, this forces them to create the output files themselves.

Reply:
In general, we should always avoid committing **dist** or **build** stuff. For GitHub users, we can publish distributions to the [release](https://github.com/jh3y/ep/releases) page so that they can directly download the distribution without clone the whole repo.

Reply:
Had completely forgot about attaching files to the release 😄  great shout!